webapp/channels/src/components/post_view/channel_intro_message/channel_intro_message.tsx

```diff
@@ -185,17 +218,26 @@ function createGMIntroMessage(channel: Channel, centeredIntro: string, profiles:
             id={channelIntroId}
             className={'channel-intro ' + centeredIntro}
         >
-            <p className='channel-intro-text'>
+            <div className='channel-intro__text'>
```

Comment:
Why are we changing these to be divs? I'd expect them to be a paragraph still

Also, changing the class name from `channel-intro-text` is going to break E2E tests because they still look for it. Ideally, I'd think the tests would look for the text of this message instead of a class name, but we should at the very least update those tests to new class name

Reply:
@hmhealey I changed this back to p - thanks for catching that